.github/workflows/cifar_benchmark.yaml

```diff
@@ -177,6 +167,25 @@ jobs:
           name: server.zip
           path: client_server/server.zip
 
+      # We need to keep this as the last step to avoid not uploading the artifacts
+      # if the step crashes
+      - name: Upload results
+        id: upload-results
+        run: |
+          # Log the json
+          cat to_upload.json | jq
+          
+          # We need to sleep to avoid log issues
+          sleep 1.
+
+          # Upload the json to the benchmark database
+          curl --fail-with-body \
```

Comment:
With this option the curl request fails if the status code is >= 400

Reply:
nice